cts/cli/regression.rules.exp

```diff
@@ -11,23 +11,17 @@ crm_rule: --check requires use of --rule=
 =#=#=#= End test: crm_rule given no rule to check - Incorrect usage (64) =#=#=#=
 * Passed: crm_rule       - crm_rule given no rule to check
 =#=#=#= Begin test: crm_rule given invalid input XML =#=#=#=
-(log_xmllib_err@xml.c:891) 	error: XML Error: Entity: line 1: parser error : Start tag expected, '<' not found
```

Comment:
Do we need this? As mentioned in the commit message, we can get this (minus the `"@xml.c:891"`) if add verbosity, but we no longer get it by default.

Reply:
This is actually what caused the CI build to fail.

`build-all-voting` succeeded, because `cts-cli` doesn't produce these `(log_xmllib_err@xml.c:891)` messages, which I removed from the test output.

However, `build-rpms` failed. The `(log_xmllib_err@xml.c:891)` messages DO appear in the output when running `make -C rpm rpm`.

I don't know what could be causing this discrepancy.

Reply:
Okay, here's the deal... Initially, I ran `cts-cli -t rules` after writing the code in this PR, and it didn't include the `log_xmllib_err` lines. So I removed them from `regression.tools.exp`. All CTS tests ran fine on my machine after that, and I pushed.

CI passed `build-all-voting` and failed `build-rpms`. I reproduced in my machine that `make -C rpm rpm` fails due to a diff in the `cts-cli` tests.

It's fixed now after I put those lines back in. It turns out there's a really strange issue with `cts-cli`, where it prints the `log_xmllib_err` lines with certain combinations of tests but not others. Likewise, another test group (`tools` IIRC) `unpack_resources` errors with some test group combinations and not with others. I filed https://projects.clusterlabs.org/T548 for this issue. The best bet is to run the full `cts-cli` suite, to get the same output as the regression.*.exp files and to update them when needed.